project5/project5.sql

```diff
@@ -0,0 +1,73 @@
+CREATE TABLE transport_condition(
+Id SERIAL PRIMARY KEY,
+name TEXT Unique NOT NULL);
+
+CREATE TABLE transport_unit(
+tail_number SERIAL PRIMARY KEY, --значения бортового номера уникальны для каждого автомобиля
+
+release_year INT NOT NULL check(release_year > 0),
```

Comment:
It seems like the release_year check could be more accurate I'd say `> 1900` or even `> 1960`